bindings/CMakeLists.txt

```
@@ -5,6 +5,10 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
 if(pyroot)
+  if (MSVC OR (NOT CMAKE_INSTALL_LIBDIR STREQUAL CMAKE_INSTALL_PYTHONDIR AND how_many_pythons GREATER 1))
```

Is this supposed to be:
```suggestion
  if ((MSVC AND NOT CMAKE_INSTALL_LIBDIR STREQUAL CMAKE_INSTALL_PYTHONDIR) OR how_many_pythons GREATER 1)
```

---

@oshadura is right that my logic is wrong for windows, but this is not it either, I think.

We need to error out if a custom `CMAKE_INSTALL_PYTHONDIR` is used on windows or with `how_many_pythons > 1`. The tricky part is that on windows the default destination for python stuff is not `CMAKE_INSTALL_LIBDIR` but `CMAKE_INSTALL_BINDIR`. I'll figure it out :smile:

---

should be fixed